### PR TITLE
Remove failing acceptance test

### PR DESCRIPTION
**Pull request by steven-sheehy**
**Detailed description**:
Acceptance tests are failing on mainnet with the below error for the `Validate Negative Association` flow:
```
2021-01-14T10:38:46.604-0600 DEBUG main c.h.m.t.e.a.c.TokenClient Create new token OHWD 
2021-01-14T10:38:46.941-0600 DEBUG main c.h.m.t.e.a.c.AbstractNetworkClient Executed transaction 0.0.950@1610642316.866285000. 
2021-01-14T10:38:50.957-0600 DEBUG main c.h.m.t.e.a.c.TokenClient Created new token 0.0.107629 
  Given I successfully create a new token                                                            # com.hedera.mirror.test.e2e.acceptance.steps.TokenFeature.createNewToken()
2021-01-14T10:38:50.959-0600 DEBUG main c.h.m.t.e.a.c.TokenClient Associate account 0.0.950 with token 0.0.107629 
2021-01-14T10:38:51.182-0600 INFO main c.h.m.t.e.a.s.TokenFeature Exception mismatch : transaction 0.0.950@1610642321.108028000 failed precheck with status INSUFFICIENT_TX_FEE 
  Then the network should observe an error associating a token "TOKEN_ALREADY_ASSOCIATED_TO_ACCOUNT" # com.hedera.mirror.test.e2e.acceptance.steps.TokenFeature.verifyTokenAssociation(java.lang.String)
      java.lang.Exception: Unexpected error code returned
	at com.hedera.mirror.test.e2e.acceptance.steps.TokenFeature.verifyTokenAssociation(TokenFeature.java:353)
	at ???.the network should observe an error associating a token "TOKEN_ALREADY_ASSOCIATED_TO_ACCOUNT"
```

Other token associates before this work fine. It's not clear why we are getting insufficient tx fee as 1_000_000_000 should be high enough for most token associates per fee calculator. However, I think it bad practice to be testing negative scenarios of HAPI as such tests don't touch any part of the mirror node. As such, I recommend we remove these two negative tests.

Another reason to remove this test is using the static payer account for token associate will eventually run into `TOKENS_PER_ACCOUNT_LIMIT_EXCEEDED` errors.

**Which issue(s) this PR fixes**:

**Special notes for your reviewer**:

**Checklist**
- [ ] Documentation added
- [x] Tests updated

### CODE REVIEW

File: hedera-mirror-test/src/test/resources/features/hts/hts.feature [hunk 1]
```diff
@@ -55,19 +55,3 @@ Feature: HTS Base Coverage Feature
         Examples:
             | httpStatusCode |
             | 200            |
-
-#    @Negative @Acceptance
```
Comment (Nana-EC) [inline hedera-mirror-test/src/test/resources/features/hts/hts.feature:59]: Removing these means the `verifyTokenCreation()` and `verifyTokenAssociation()` are now orphaned methods. Please verify and if so they should be removed.
Unless you plan to tie them into a new test.

**Comment by ijungmann** [comment]
Those two verification methods in `TokenFeature` are only used here, probably should remove those too.

**Review by Nana-EC — CHANGES_REQUESTED** [review]
Works for me, just some dead code now that can be removed.

File: hedera-mirror-test/src/test/resources/features/hts/hts.feature (hunk 1 above)
Comment (steven-sheehy) [reply hedera-mirror-test/src/test/resources/features/hts/hts.feature:59]: Good catch. Removed

**Review by ijungmann — APPROVED** [review]

**Review by Nana-EC — APPROVED** [review]
LGTM

**Pull request merged**